tests/integration/test_metdata_cache_memory_leak/test.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance("node", user_configs=["configs/users.xml"])
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_metadata_cache_memleak(started_cluster):
+    if not node.is_built_with_address_sanitizer():
+        pytest.skip("Test is only valid with address(leak) sanitizer build")
```

Comment:
I guess we can remove this, and leave only a comment that the problem will be shown only under ASan

Reply:
Though maybe it is good to leave it this way to make more explicit, comments are more likely can be ignored then `pytest.skip`